### DISCUSSION TITLE
[SOLVED] Getting fetch error with Prismic integration

### DISCUSSION THREAD

**Discussion by ggluta** [vercel/next.js#34024] [Help] [answered]
Hi there! I've just created a dumb demo project so I can test out the integration with Prismic CMS. I followed all the steps from [here](https://github.com/vercel/next.js/tree/canary/examples/cms-prismic#step-5-set-up-environment-variables). All nice and dandy until step 6. When I run `yarn dev`, I get this weird error:

```
error - FetchError: request to https://hello-prismic-nextjs.prismic.io.cdn.prismic.io/api/v2?access_token=xxx_my_private_token_xxx failed, reason: Hostname/IP does not match certificate's altnames: Host: hello-prismic-nextjs.prismic.io.cdn.prismic.io. is not in the cert's altnames: DNS:*.cdn.prismic.io
```

TLDR;
Error shouts `reason: Hostname/IP does not match certificate's altnames`

Usually, I figure out where it cracked but I couldn't seem to get enough answers on Google when pairing the search with `prismic` keyword. Hope you guys can help me figure out what happened. Could it be a config mess up? Thx in advance! 🙏

**Comment by ggluta** [comment, marked as answer]
Okay, after some more debugging and head-scratching, I realized where the dumb mistake was. The `PRISMIC_REPOSITORY_NAME` should be the name only. I got confused by the README mentioning the URL. So
`hello-prismic-nextjs.prismic.io.cdn.prismic.io` becomes `hello-prismic-nextjs.cdn.prismic.io`.

This thread can be archived or deleted. Thanks!
